pkg/phases/consul/tasks.go

```diff
@@ -0,0 +1,97 @@
+package consul
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/flanksource/commons/utils"
+	"github.com/moshloop/platform-cli/pkg/k8s"
+	"github.com/moshloop/platform-cli/pkg/platform"
+)
+
+type BackupRestore struct {
+	Name      string
+	Namespace string
+	client    *k8s.Client
+	platform  *platform.Platform
```

Comment:
```suggestion
	Name      string
	Namespace string
        *platform.Platform
```

If you make platform an embedded struct, then you can skip all the `b.client.Apply` and just use `b.Apply`